packages/web/src/components/profile-progress/components/ProfileCompletionTooltip.tsx

```diff
@@ -18,15 +20,27 @@ type TooltipContentProps = {
 const TooltipContent = ({ completionStages }: TooltipContentProps) => {
   const completionPercentage = getPercentageComplete(completionStages).toFixed()
   const strings = makeStrings(Number(completionPercentage))
+  const { color } = useTheme()
 
   return (
-    <div className={styles.content}>
-      <div className={styles.header}>{strings.completionPercentage}</div>
-      <TaskCompletionList
-        className={styles.list}
-        completionStages={completionStages}
-      />
-    </div>
+    <Flex direction='column'>
+      <Flex
+        p='l'
+        color='white'
```

Comment:
don't want to use a harmony color here and below in text?

Reply:
woops, removed